crd-generator/src/main/java/io/strimzi/crdgenerator/annotations/DescriptionFile.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.crdgenerator.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface DescriptionFile {
```

Comment:
We should document what `.adoc` file this annotation implies.

Reply:
You mean ... document in this class as JavaDoc? Or did you had some other way in mind?

Reply:
Yeah, just as javadoc